news/2018-07-20-the-qat-gazette-june-2018.md

```diff
@@ -0,0 +1,98 @@
+
+Still different from the BN score of pre-2014 days, this will take into account a more subjective and up-to-date view of disqualifications, so as not to punish Beatmap Nominators unnecessarily harshly for what may be minor or non-impactful changes.
+
+### New Beatmap Nominator Promotions
+
+After three agonizing weeks of waiting, we're pleased to welcome nine new Beatmap Nominators to our big happy family of nearly one hundred members. Please give a warm welcome to the following people, whose friendly faces will be in the discussion threads of your favorite Qualified and Ranked beatmaps:
+
+[_Stan](https://osu.ppy.sh/users/1653229), [DE-CADE](https://osu.ppy.sh/users/3734394), [DoNotMess](https://osu.ppy.sh/users/1596318), [Fii](https://osu.ppy.sh/users/3922569), [Lenfried-](https://osu.ppy.sh/users/5314573), [Nifty](https://osu.ppy.sh/users/4956097), [qoot8123](https://osu.ppy.sh/users/766371), [Unpredictable](https://osu.ppy.sh/users/7560872) and [Yumeno Himiko](https://osu.ppy.sh/users/1806962)
+
+Many congratulations on becoming Beatmap Nominators, everyone – we hope you enjoy your time with us!
+
+### Beatmap Nominator Retirements
+
+As always though, some of our hard working members have departed the team to pursue other goals outside of the focus of the Beatmap Nominators. Please give thanks to the following for their hard work and contribution to the wonderful mapping and modding community:
+
+[AyonokoRin](https://osu.ppy.sh/users/5376866), [dsco](https://osu.ppy.sh/users/4330511), [celerih](https://osu.ppy.sh/users/4696296), [frukoyurdakul](https://osu.ppy.sh/users/7612550), [Hailie](https://osu.ppy.sh/users/5226970) and [Kagetsu](https://osu.ppy.sh/users/6203841) 
+
+Their unique perspectives and courage to explore the frontiers of mapping have beaten a path for others to follow and expand. Thank you for your contributions and good luck wherever you go next.
+
+## Within the Quality Assurance Team
+
+June was a fairly hectic month for the Quality Assurance Team. We had our second ever public QAT meeting and from that, we emerged with a new vision and direction for the team, along with a slight restructuring of how responsibilities are shared.
+
+As a result, we said goodbye to some old and long serving friends, and welcomed aboard a new member to the team. It's quite a lot to keep track of, but we'll head through one thing at a time, so please read on to stay on top of the latest changes and news!
+
+### Public QAT Meeting
+
+On June 17th, we had our second public QAT meeting in the osu!dev server, after the success of the first back in April. This meeting was advertised much more in advance, and this was reflected by a considerably higher turnout with more community involved discussion.
```

Comment:
`osu!dev server` -> `` `#modding` channel ``

Reply:
The dev server hasn't been linked yet, it makes no sense to discuss the specific channel just yet